zboxcore/sdk/writemarker_mutex.go

```diff
@@ -39,15 +41,26 @@ type WMLockResult struct {
 // WriteMarkerMutex blobber WriteMarkerMutex client
 type WriteMarkerMutex struct {
 	mutex          sync.Mutex
+	zbox           *sdks.ZBox
 	allocationObj  *Allocation
-	lockedBlobbers map[string]bool
+	lockedBlobbers map[int]bool
```

Comment:
Using index as key here might cause unexpected behavior if the blobber list is changed (although we should not keep the blobbers position in the list unchanged). Anyway, I would recommand to keep the blobber url as key to avoid potential issues.

Reply:
fixed with blobber.id